assets/js/blocks/cart/inner-blocks/cart-cross-sells-block/frontend.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { useStoreCart } from '@woocommerce/base-context/hooks';
+
+interface Props {
+	children?: JSX.Element | JSX.Element[];
+	className?: string;
+}
+
+const FrontendBlock = ( {
+	children,
+	className = '',
+}: Props ): JSX.Element | null => {
+	const { crossSellsProducts, cartIsLoading } = useStoreCart();
```

Comment:
Let's investigate why `cartIsLoading` is always false for our block, it could be because it's an inner block and it's only rendered when this is finished, if so we could remove this.

Reply:
That would be great, if you could assist me troubleshooting this one.